.cicd-cli.py

```
@@ -137,10 +138,10 @@ def  _fix_version(parsed_version: str, parsed_epoch: int):
     if argument default is used and it is semver it is likely 'today'. Use
     current day in this case.
     """
-    pattern = re.compile(r'^[\d\.]+$')
+    pattern = re.compile(r'^[\d.]+$')
```

I believe we are specifically looking for \d\. (number followed by period) and not \d. (number followed by any character).

---

This is inside a set [], so the dot is literal. The backslash would normally start an escape sequence, but `\.` is not a valid one (inside a set!) so the slash is silently ignored and misleading.